source/Engine.cpp

```diff
@@ -1461,10 +1461,26 @@ void Engine::CalculateStep()
 
 	// Keep track of the flagship to see if it jumps or enters a wormhole this turn.
 	const Ship *flagship = player.Flagship();
+	bool flagshipWasUntargetable = (flagship && !flagship->IsTargetable());
```

Comment:
isnt this the exact same value as:
bool flagshipIsUntargetable = (flagship && !flagship->IsTargetable());

two other things:
 1. this can be const
 2. you dont need the brackets

Reply:
This one is before the MoveShip call. Calling MoveShip may have changed the value of flagship->IsTargetable().

Reply:
ye I just noticed that :sweat: